.github/workflows/load-tests.yml

```diff
@@ -126,12 +143,40 @@ jobs:
       - name: Setup K6
         if: ${{ steps.should.outputs.RUN == 'true' }}
         uses: grafana/setup-k6-action@v1
-      - name: Run k6 Tests
-        if: ${{ steps.should.outputs.RUN == 'true' }}
+
+      - name: Run k6 Tests - PR Mode (Light Load Only)
+        if: ${{ steps.should.outputs.RUN == 'true' && steps.test-type.outputs.test_type == 'pr' }}
         run: |
           for test_file in apps/${{ matrix.service }}/k6-test/*.k6.js; do
-            k6 run --no-color --quiet --no-usage-report "$test_file" || exit 1
+            # Skip stress tests for PRs
```

Comment:
were you able to test these actions locally to make sure they run at the right time?  Something like https://github.com/bahdotsh/wrkflw/tree/v0.7.0 might help